core/iwasm/interpreter/wasm_loader.c

```diff
@@ -3503,7 +3503,7 @@ check_wasi_abi_compatibility(const WASMModule *module, bool main_module,
                 ->func_type;
         if (func_type->param_count || func_type->result_count) {
             set_error_buf(error_buf, error_buf_size,
-                          "The builtin _start() is with a wrong signature");
+                          "the signature of builtin _start() func is wrong");
```

Comment:
feel it is ok either "__start function" or "__start()". Since "()" means a function.

Reply:
Done